internal/util/version.go

```diff
@@ -50,7 +50,7 @@ func readVersion() (Version, error) {
 		Api:     UNKNOWN_API,
 	}
 
-	versionPath := ShouldAbs(filepath.Join(ShouldGetThisDir(), "..", "..", VERSION_FILENAME))
+	versionPath := ShouldAbs(filepath.Join(ShouldGetThisDir(), "..", "..", "resources", VERSION_FILENAME))
```

Comment:
WTF?
"What the funk?"

You make GetResourcesDir() and then don't use it?
Maybe it belongs here instead.